### PR TITLE
feat(wallet-platform): add openApi doc for transaction with emergency…

### PR DESCRIPTION
**Pull request by nicolasbellanich-simtlix**
Add openApi doc for transaction with emergency param

stlx-16868

### CODE REVIEW

File: modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { BitGo, Wallet } from 'bitgo';
```
Comment (mmcshinsky-bitgo) [inline modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts:1]: Missing title comments. See other examples.

File: modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+const coin = 'talgo:16026733';
+    
+// this can be found on test.bitgo.com in the URL after clicking on a wallet
+// https://test.bitgo.com/enterprise/XXXXXXXXX/coin/talgo/YYYYY/transactions
+const walletId = 'your wallet id';
+    
+// this is your wallet passphrase, which could be different than your login credentials
+const walletPassphrase = 'set your wallet passphrase here';
+    
+// this will need to be a real OTP code on production
```
Comment (mmcshinsky-bitgo) [inline modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts:17]: Possible disclosure as optional?

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]

**Review by JulioAchaval-SimTLiX — APPROVED** [review]

File: modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts (hunk 1 above)
Comment (mmcshinsky-bitgo) [reply modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts:7]: Let's add `Copyright 2020, BitGo, Inc.  All Rights Reserved.` and we're good to go (for consistency).

File: modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    recipients: [
+      {
+        amount: '0',
+        address: address,
+
+      },
+    ],
+    feeRate: 1,
+    walletPassphrase: walletPassphrase,
+    emergency: true,
```
Comment (tylerlevine) [inline modules/bitgo/example/ts/algo/transaction-with-emergency-param.ts:82]: I'm curious why we didn't add a `validityWindow` param or something more useful than a boolean for either 1 or 2 hours, but that ship has sailed at this point.

Thanks for adding the example 👍

**Review by tylerlevine — DISMISSED** [review]
also one other note - there's no OpenAPI doc in here so I'm not sure why that's being mentioned in the commit message

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**